lib/twingly/http.rb

```diff
@@ -61,6 +61,10 @@ def post(url, body:, headers: {})
 
       private
 
+      def default_logger
+        Logger.new("/dev/null")
```

Comment:
There's a [`File::NULL` constant](https://ruby-doc.org/core-2.7.2/File/Constants.html) which just contains the string `"/dev/null"`. TIL :)

Reply:
Guess it could be nice to use :)

Reply:
Cool! I replaced the string in 7613cec